org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/output/FoldResult.java

```diff
@@ -0,0 +1,63 @@
+package org.metaborg.spoofax.shell.output;
+
+import java.util.List;
+
+import org.metaborg.core.source.ISourceRegion;
+import org.metaborg.core.style.IRegionStyle;
+import org.spoofax.interpreter.terms.IStrategoTerm;
+
+/**
+ * Represents a result that is a term and foldable regions.
+ */
+public class FoldResult implements IResult {
```

Comment:
Should it not maybe inherit from an `ISpoofaxTermResult`, seeing that this result also has a term?